crates/env/src/engine/on_chain/impls.rs

```diff
@@ -39,7 +39,6 @@ use crate::{
     Clear,
     EnvBackend,
```

Comment:
Please remove everything that is associated to the `on_chain` environment or dispatch between on- and off-chain environments. Has absolutely no meaning in this crate.